xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -4932,12 +4947,26 @@ void CVideoPlayer::UpdateContent()
 void CVideoPlayer::UpdateContentState()
 {
   CSingleLock lock(m_content.m_section);
+
   m_content.m_videoIndex = m_SelectionStreams.TypeIndexOf(STREAM_VIDEO, m_CurrentVideo.source,
                                                       m_CurrentVideo.demuxerId, m_CurrentVideo.id);
   m_content.m_audioIndex = m_SelectionStreams.TypeIndexOf(STREAM_AUDIO, m_CurrentAudio.source,
                                                       m_CurrentAudio.demuxerId, m_CurrentAudio.id);
   m_content.m_subtitleIndex = m_SelectionStreams.TypeIndexOf(STREAM_SUBTITLE, m_CurrentSubtitle.source,
                                                          m_CurrentSubtitle.demuxerId, m_CurrentSubtitle.id);
+
+  if (m_content.m_videoIndex == -1 && m_content.m_audioIndex == -1)
+  {
```

Comment:
This is missing the condition "&& m_pInputStream->IsStreamType(DVDSTREAM_TYPE_DVD)". Probably causing the reported crash.

Reply:
PR here https://github.com/xbmc/xbmc/pull/19645